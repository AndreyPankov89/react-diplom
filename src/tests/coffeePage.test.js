import React from 'react';
import CoffeePage from '../components/pages/coffeePage';
import {shallow} from 'enzyme';

describe('Testing <coffeePage/>', () => {
    const item = shallow(<CoffeePage/>);
    describe('testing shap & state ', () =>{
        it('BestItem have rendered correctly', () => {
            expect(item).toMatchSnapshot();
            
        })
    });
})