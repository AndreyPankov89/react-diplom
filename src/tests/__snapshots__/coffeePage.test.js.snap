// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <coffeePage/> testing shap & state  BestItem have rendered correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <CoffeePage />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": <section
        className="shop"
      >
        <div
          className="container"
        >
          <div
            className="row"
          >
            <div
              className="col-lg-4 offset-2"
            >
              <img
                alt="girl"
                className="shop__girl"
                src="/img/coffee_girl.jpg"
              />
            </div>
            <div
              className="col-lg-4"
            >
              <div
                className="title"
              >
                About our beans
              </div>
              <img
                alt="Beans logo"
                className="beanslogo"
                src="/logo/Beans_logo_dark.svg"
              />
              <div
                className="shop__text"
              >
                Extremity sweetness difficult behaviour he of. On disposal of as landlord horrible.
                <br />
                <br />
                Afraid at highly months do things on at. Situation recommend objection do intention
                <br />
                so questions. 
                <br />
                As greatly removed calling pleased improve an. Last ask him cold feel
                <br />
                met spot shy want. Children me laughing we prospect answered followed. At it went
                <br />
                is song that held help face.
              </div>
            </div>
          </div>
          <div
            className="line"
          />
          <ShopItems
            content="coffee"
          />
        </div>
      </section>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="container"
        >
          <div
            className="row"
          >
            <div
              className="col-lg-4 offset-2"
            >
              <img
                alt="girl"
                className="shop__girl"
                src="/img/coffee_girl.jpg"
              />
            </div>
            <div
              className="col-lg-4"
            >
              <div
                className="title"
              >
                About our beans
              </div>
              <img
                alt="Beans logo"
                className="beanslogo"
                src="/logo/Beans_logo_dark.svg"
              />
              <div
                className="shop__text"
              >
                Extremity sweetness difficult behaviour he of. On disposal of as landlord horrible.
                <br />
                <br />
                Afraid at highly months do things on at. Situation recommend objection do intention
                <br />
                so questions. 
                <br />
                As greatly removed calling pleased improve an. Last ask him cold feel
                <br />
                met spot shy want. Children me laughing we prospect answered followed. At it went
                <br />
                is song that held help face.
              </div>
            </div>
          </div>
          <div
            className="line"
          />
          <ShopItems
            content="coffee"
          />
        </div>,
        "className": "shop",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="row"
            >
              <div
                className="col-lg-4 offset-2"
              >
                <img
                  alt="girl"
                  className="shop__girl"
                  src="/img/coffee_girl.jpg"
                />
              </div>
              <div
                className="col-lg-4"
              >
                <div
                  className="title"
                >
                  About our beans
                </div>
                <img
                  alt="Beans logo"
                  className="beanslogo"
                  src="/logo/Beans_logo_dark.svg"
                />
                <div
                  className="shop__text"
                >
                  Extremity sweetness difficult behaviour he of. On disposal of as landlord horrible.
                  <br />
                  <br />
                  Afraid at highly months do things on at. Situation recommend objection do intention
                  <br />
                  so questions. 
                  <br />
                  As greatly removed calling pleased improve an. Last ask him cold feel
                  <br />
                  met spot shy want. Children me laughing we prospect answered followed. At it went
                  <br />
                  is song that held help face.
                </div>
              </div>
            </div>,
            <div
              className="line"
            />,
            <ShopItems
              content="coffee"
            />,
          ],
          "className": "container",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <div
                  className="col-lg-4 offset-2"
                >
                  <img
                    alt="girl"
                    className="shop__girl"
                    src="/img/coffee_girl.jpg"
                  />
                </div>,
                <div
                  className="col-lg-4"
                >
                  <div
                    className="title"
                  >
                    About our beans
                  </div>
                  <img
                    alt="Beans logo"
                    className="beanslogo"
                    src="/logo/Beans_logo_dark.svg"
                  />
                  <div
                    className="shop__text"
                  >
                    Extremity sweetness difficult behaviour he of. On disposal of as landlord horrible.
                    <br />
                    <br />
                    Afraid at highly months do things on at. Situation recommend objection do intention
                    <br />
                    so questions. 
                    <br />
                    As greatly removed calling pleased improve an. Last ask him cold feel
                    <br />
                    met spot shy want. Children me laughing we prospect answered followed. At it went
                    <br />
                    is song that held help face.
                  </div>
                </div>,
              ],
              "className": "row",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <img
                    alt="girl"
                    className="shop__girl"
                    src="/img/coffee_girl.jpg"
                  />,
                  "className": "col-lg-4 offset-2",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "girl",
                    "className": "shop__girl",
                    "src": "/img/coffee_girl.jpg",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
                "type": "div",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <div
                      className="title"
                    >
                      About our beans
                    </div>,
                    <img
                      alt="Beans logo"
                      className="beanslogo"
                      src="/logo/Beans_logo_dark.svg"
                    />,
                    <div
                      className="shop__text"
                    >
                      Extremity sweetness difficult behaviour he of. On disposal of as landlord horrible.
                      <br />
                      <br />
                      Afraid at highly months do things on at. Situation recommend objection do intention
                      <br />
                      so questions. 
                      <br />
                      As greatly removed calling pleased improve an. Last ask him cold feel
                      <br />
                      met spot shy want. Children me laughing we prospect answered followed. At it went
                      <br />
                      is song that held help face.
                    </div>,
                  ],
                  "className": "col-lg-4",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "About our beans",
                      "className": "title",
                    },
                    "ref": null,
                    "rendered": "About our beans",
                    "type": "div",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "alt": "Beans logo",
                      "className": "beanslogo",
                      "src": "/logo/Beans_logo_dark.svg",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": "img",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": Array [
                        "Extremity sweetness difficult behaviour he of. On disposal of as landlord horrible.",
                        <br />,
                        <br />,
                        "Afraid at highly months do things on at. Situation recommend objection do intention",
                        <br />,
                        "so questions. ",
                        <br />,
                        "As greatly removed calling pleased improve an. Last ask him cold feel",
                        <br />,
                        "met spot shy want. Children me laughing we prospect answered followed. At it went",
                        <br />,
                        "is song that held help face.",
                      ],
                      "className": "shop__text",
                    },
                    "ref": null,
                    "rendered": Array [
                      "Extremity sweetness difficult behaviour he of. On disposal of as landlord horrible.",
                      Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "host",
                        "props": Object {},
                        "ref": null,
                        "rendered": null,
                        "type": "br",
                      },
                      Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "host",
                        "props": Object {},
                        "ref": null,
                        "rendered": null,
                        "type": "br",
                      },
                      "Afraid at highly months do things on at. Situation recommend objection do intention",
                      Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "host",
                        "props": Object {},
                        "ref": null,
                        "rendered": null,
                        "type": "br",
                      },
                      "so questions. ",
                      Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "host",
                        "props": Object {},
                        "ref": null,
                        "rendered": null,
                        "type": "br",
                      },
                      "As greatly removed calling pleased improve an. Last ask him cold feel",
                      Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "host",
                        "props": Object {},
                        "ref": null,
                        "rendered": null,
                        "type": "br",
                      },
                      "met spot shy want. Children me laughing we prospect answered followed. At it went",
                      Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "host",
                        "props": Object {},
                        "ref": null,
                        "rendered": null,
                        "type": "br",
                      },
                      "is song that held help face.",
                    ],
                    "type": "div",
                  },
                ],
                "type": "div",
              },
            ],
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "line",
            },
            "ref": null,
            "rendered": null,
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "content": "coffee",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
        ],
        "type": "div",
      },
      "type": "section",
    },
    "type": Symbol(react.fragment),
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": <section
          className="shop"
        >
          <div
            className="container"
          >
            <div
              className="row"
            >
              <div
                className="col-lg-4 offset-2"
              >
                <img
                  alt="girl"
                  className="shop__girl"
                  src="/img/coffee_girl.jpg"
                />
              </div>
              <div
                className="col-lg-4"
              >
                <div
                  className="title"
                >
                  About our beans
                </div>
                <img
                  alt="Beans logo"
                  className="beanslogo"
                  src="/logo/Beans_logo_dark.svg"
                />
                <div
                  className="shop__text"
                >
                  Extremity sweetness difficult behaviour he of. On disposal of as landlord horrible.
                  <br />
                  <br />
                  Afraid at highly months do things on at. Situation recommend objection do intention
                  <br />
                  so questions. 
                  <br />
                  As greatly removed calling pleased improve an. Last ask him cold feel
                  <br />
                  met spot shy want. Children me laughing we prospect answered followed. At it went
                  <br />
                  is song that held help face.
                </div>
              </div>
            </div>
            <div
              className="line"
            />
            <ShopItems
              content="coffee"
            />
          </div>
        </section>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="container"
          >
            <div
              className="row"
            >
              <div
                className="col-lg-4 offset-2"
              >
                <img
                  alt="girl"
                  className="shop__girl"
                  src="/img/coffee_girl.jpg"
                />
              </div>
              <div
                className="col-lg-4"
              >
                <div
                  className="title"
                >
                  About our beans
                </div>
                <img
                  alt="Beans logo"
                  className="beanslogo"
                  src="/logo/Beans_logo_dark.svg"
                />
                <div
                  className="shop__text"
                >
                  Extremity sweetness difficult behaviour he of. On disposal of as landlord horrible.
                  <br />
                  <br />
                  Afraid at highly months do things on at. Situation recommend objection do intention
                  <br />
                  so questions. 
                  <br />
                  As greatly removed calling pleased improve an. Last ask him cold feel
                  <br />
                  met spot shy want. Children me laughing we prospect answered followed. At it went
                  <br />
                  is song that held help face.
                </div>
              </div>
            </div>
            <div
              className="line"
            />
            <ShopItems
              content="coffee"
            />
          </div>,
          "className": "shop",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div
                className="row"
              >
                <div
                  className="col-lg-4 offset-2"
                >
                  <img
                    alt="girl"
                    className="shop__girl"
                    src="/img/coffee_girl.jpg"
                  />
                </div>
                <div
                  className="col-lg-4"
                >
                  <div
                    className="title"
                  >
                    About our beans
                  </div>
                  <img
                    alt="Beans logo"
                    className="beanslogo"
                    src="/logo/Beans_logo_dark.svg"
                  />
                  <div
                    className="shop__text"
                  >
                    Extremity sweetness difficult behaviour he of. On disposal of as landlord horrible.
                    <br />
                    <br />
                    Afraid at highly months do things on at. Situation recommend objection do intention
                    <br />
                    so questions. 
                    <br />
                    As greatly removed calling pleased improve an. Last ask him cold feel
                    <br />
                    met spot shy want. Children me laughing we prospect answered followed. At it went
                    <br />
                    is song that held help face.
                  </div>
                </div>
              </div>,
              <div
                className="line"
              />,
              <ShopItems
                content="coffee"
              />,
            ],
            "className": "container",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <div
                    className="col-lg-4 offset-2"
                  >
                    <img
                      alt="girl"
                      className="shop__girl"
                      src="/img/coffee_girl.jpg"
                    />
                  </div>,
                  <div
                    className="col-lg-4"
                  >
                    <div
                      className="title"
                    >
                      About our beans
                    </div>
                    <img
                      alt="Beans logo"
                      className="beanslogo"
                      src="/logo/Beans_logo_dark.svg"
                    />
                    <div
                      className="shop__text"
                    >
                      Extremity sweetness difficult behaviour he of. On disposal of as landlord horrible.
                      <br />
                      <br />
                      Afraid at highly months do things on at. Situation recommend objection do intention
                      <br />
                      so questions. 
                      <br />
                      As greatly removed calling pleased improve an. Last ask him cold feel
                      <br />
                      met spot shy want. Children me laughing we prospect answered followed. At it went
                      <br />
                      is song that held help face.
                    </div>
                  </div>,
                ],
                "className": "row",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <img
                      alt="girl"
                      className="shop__girl"
                      src="/img/coffee_girl.jpg"
                    />,
                    "className": "col-lg-4 offset-2",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "alt": "girl",
                      "className": "shop__girl",
                      "src": "/img/coffee_girl.jpg",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": "img",
                  },
                  "type": "div",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      <div
                        className="title"
                      >
                        About our beans
                      </div>,
                      <img
                        alt="Beans logo"
                        className="beanslogo"
                        src="/logo/Beans_logo_dark.svg"
                      />,
                      <div
                        className="shop__text"
                      >
                        Extremity sweetness difficult behaviour he of. On disposal of as landlord horrible.
                        <br />
                        <br />
                        Afraid at highly months do things on at. Situation recommend objection do intention
                        <br />
                        so questions. 
                        <br />
                        As greatly removed calling pleased improve an. Last ask him cold feel
                        <br />
                        met spot shy want. Children me laughing we prospect answered followed. At it went
                        <br />
                        is song that held help face.
                      </div>,
                    ],
                    "className": "col-lg-4",
                  },
                  "ref": null,
                  "rendered": Array [
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "About our beans",
                        "className": "title",
                      },
                      "ref": null,
                      "rendered": "About our beans",
                      "type": "div",
                    },
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "alt": "Beans logo",
                        "className": "beanslogo",
                        "src": "/logo/Beans_logo_dark.svg",
                      },
                      "ref": null,
                      "rendered": null,
                      "type": "img",
                    },
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": Array [
                          "Extremity sweetness difficult behaviour he of. On disposal of as landlord horrible.",
                          <br />,
                          <br />,
                          "Afraid at highly months do things on at. Situation recommend objection do intention",
                          <br />,
                          "so questions. ",
                          <br />,
                          "As greatly removed calling pleased improve an. Last ask him cold feel",
                          <br />,
                          "met spot shy want. Children me laughing we prospect answered followed. At it went",
                          <br />,
                          "is song that held help face.",
                        ],
                        "className": "shop__text",
                      },
                      "ref": null,
                      "rendered": Array [
                        "Extremity sweetness difficult behaviour he of. On disposal of as landlord horrible.",
                        Object {
                          "instance": null,
                          "key": undefined,
                          "nodeType": "host",
                          "props": Object {},
                          "ref": null,
                          "rendered": null,
                          "type": "br",
                        },
                        Object {
                          "instance": null,
                          "key": undefined,
                          "nodeType": "host",
                          "props": Object {},
                          "ref": null,
                          "rendered": null,
                          "type": "br",
                        },
                        "Afraid at highly months do things on at. Situation recommend objection do intention",
                        Object {
                          "instance": null,
                          "key": undefined,
                          "nodeType": "host",
                          "props": Object {},
                          "ref": null,
                          "rendered": null,
                          "type": "br",
                        },
                        "so questions. ",
                        Object {
                          "instance": null,
                          "key": undefined,
                          "nodeType": "host",
                          "props": Object {},
                          "ref": null,
                          "rendered": null,
                          "type": "br",
                        },
                        "As greatly removed calling pleased improve an. Last ask him cold feel",
                        Object {
                          "instance": null,
                          "key": undefined,
                          "nodeType": "host",
                          "props": Object {},
                          "ref": null,
                          "rendered": null,
                          "type": "br",
                        },
                        "met spot shy want. Children me laughing we prospect answered followed. At it went",
                        Object {
                          "instance": null,
                          "key": undefined,
                          "nodeType": "host",
                          "props": Object {},
                          "ref": null,
                          "rendered": null,
                          "type": "br",
                        },
                        "is song that held help face.",
                      ],
                      "type": "div",
                    },
                  ],
                  "type": "div",
                },
              ],
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "className": "line",
              },
              "ref": null,
              "rendered": null,
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "content": "coffee",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
          ],
          "type": "div",
        },
        "type": "section",
      },
      "type": Symbol(react.fragment),
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
